src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding: 0 16px 16px;
+  margin-right: 50px;
+  margin-top: 48px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__img {
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    color: #{$main-accent-color};
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #{$secondary-color};
+    margin-top: 4px;
+    width: 100%;
+  }
+
+  &__feedback {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #{$main-accent-color};
+    display: flex;
+    justify-self: center;
+    align-self: center;
+  }
+
+  &__rating {
+    display: flex;
+    margin-top: 16px;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__cost {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #{$secondary-color};
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #{$main-accent-color};
+  }
+
+  &__buy {
+    display: flex;
+    margin-top: 16px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    background-color: #{$blue-accent};
+    border: 1px solid white;
+    border-radius: 5px;
+    width: 166px;
+    height: 40px;
+    text-transform: uppercase;
+    transition: background-color 0.5s ease, color 0.5s ease, border 0.5s ease;
+
+    &:hover {
+      background-color: #fff;
+      border-color: #{$blue-accent};
+      color: #{$blue-accent};
+      transition: background-color 0.5s ease, color 0.5s ease, border 0.5s ease;
```

Comment:
I think that it is unnecessary if you have written the same styles in the button itself.
Look at this moment:
[Transition](https://youtu.be/seH3gRxNh_s?t=542)